src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -389,6 +389,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                     recursively when uploading from local file system.')
         c.argument('put_md5', action='store_true', help='Create an MD5 hash of each file, and save the hash \
                     as the Content-MD5 property of the destination blob/file.Only available when uploading.')
+        c.argument('blob_type', validator=validate_blob_type,
```

Comment:
Any difference between get_enum_type(get_blob_types())) and it?

Reply:
enum values are different. For azcopy, blob types must be one of "BlockBlob", "PageBlob", "AppendBlob".